SlicerCART/documentation/installation.md

```diff
@@ -0,0 +1,62 @@
+# SlicerCART Installation
+
+## Requirements
+
+* MacOS Sonoma or Sequoia (15.0.1) is recommended 
```

Comment:
that's way too specific. What if 15.0.2 comes out? Can I use it? likely so. I suggest another wording, be more general to other distros (see eg how we do it in SCT).